integration_tests/src/main/python/join_test.py

```diff
@@ -128,30 +121,32 @@ def join_batch_size_test_params(*args):
 
 @ignore_order(local=True)
 @pytest.mark.parametrize('join_type', ['Left', 'Inner', 'LeftSemi', 'LeftAnti'], ids=idfn)
-def test_right_broadcast_nested_loop_join_without_condition_empty(join_type):
+@pytest.mark.parametrize("aqe_enabled", ["true", "false"], ids=idfn)
+def test_right_broadcast_nested_loop_join_without_condition_empty(join_type, aqe_enabled):
     def do_join(spark):
         left, right = create_df(spark, long_gen, 50, 0)
         return left.join(broadcast(right), how=join_type)
-    assert_gpu_and_cpu_are_equal_collect(do_join)
+    assert_gpu_and_cpu_are_equal_collect(do_join, conf={ "spark.sql.adaptive.enabled": aqe_enabled, "spark.rapids.sql.debug.logTransformations": aqe_enabled })
```

Comment:
Looks like the debug.logTransformations setting here is leftover debugging code?  Also applies to the other cases of this below.